### PR TITLE
mark free accessible articls as lf https://github.com/ubtue/DatenProbleme/issues/1462

### PR DESCRIPTION
**Pull request by socheres**

### CODE REVIEW

File: ubtue_Brill.js [hunk 1]
```diff
@@ -109,7 +109,9 @@ function postProcess(doc, item) {
 	// mark articles as "LF" (MARC=856 |z|kostenfrei), that are published as open access	
 	let openAccessTag = text(doc, '.has-license span');
 	if (openAccessTag && openAccessTag.match(/open\s+access/gi)) item.notes.push('LF:');
-  
+  // mark articles as "LF" (MARC=856 |z|kostenfrei), that are free accessible e.g. conference report 10.30965/25890433-04902001 
+	let freeAccess = text(doc, '.color-access-free');Z.debug(freeAccess)
```
Comment (jriedl) [inline ubtue_Brill.js:113]: Please remove the debug message.

File: ubtue_Brill.js [hunk 2]
```diff
@@ -109,7 +109,9 @@ function postProcess(doc, item) {
...
+	if (freeAccess && freeAccess.match(/(free|freier)\s+(access|zugang)/gi)) item.notes.push('LF:');
```
Comment (jriedl) [inline ubtue_Brill.js:114]: Cf. line 111: Make sure `LF:` is inserted only once by using a `if .. else` construct?

Comment (socheres) [reply ubtue_Brill.js:114]: yes, it is either open access or free access, therefore "LF:" is only assigned once

File: ubtue_Brill.js (hunk 1 above)
Comment (socheres) [reply ubtue_Brill.js]: removed

File: ubtue_Brill.js (hunk 2 above)
Comment (jriedl) [reply ubtue_Brill.js:114]: Yes, but more cases may occur in the future. Will merge it anyway.

**Pull request merged**